molecule/mysql_hardening/verify_additional_tests.yml

```
@@ -0,0 +1,36 @@
+---
+- name: check which mysql version is used
+  community.mysql.mysql_info:
+    filter: version
+    login_unix_socket: "{{ login_unix_socket | default(omit) }}"
+  register: mysql_version
+
+- name: get all users that have no password or authentication_string on MySQL version >= 5.7.6
+  community.mysql.mysql_query:
+    query:
+      - select user,host,password from mysql.user where (length(password)=0 or password="") and (length(authentication_string)=0 or authentication_string="") and user NOT IN ('mysql.sys', 'mysqlxsys', 'mariadb.sys');
```

```suggestion
      - SELECT user, host, password
        FROM mysql.user
        WHERE (length(password)=0 OR password="")
        AND (length(authentication_string)=0 OR authentication_string="")
        AND USER NOT IN ('mysql.sys', 'mysqlxsys', 'mariadb.sys');
```
I usually like to write SQL codewords in uppercase and split statements in separate lines, putting the fieldlist, tables and conditions on separate lines. (like in the suggestion)

I think this style improves readability, do you think we should apply this to all relevant queries?

PS: a quick online formatter can be found at https://sqlformat.org/